documents/decimal.go

```diff
@@ -88,23 +93,29 @@ func (d *Decimal) Bytes() (decimal []byte, err error) {
 	s := d.dec.Abs().String()
 	sd := strings.Split(s, ".")
 
-	fraction := make([]byte, maxFractionByteLength)
+	v := sd[0]
+	fraction := strings.Repeat("0", decimalPrecision)
 	if len(sd) == 2 {
-		fraction, err = byteutils.IntBytesFromString(sd[1])
-		if err != nil {
-			return nil, err
+		// add extra zeros if precision is not max
```

Comment:
Is it possible that the user inputs a `,` instead of `.`? Does the decimal lib support it? Do we we need to fail in that case?

Reply:
yeah, we will fail then. We decided to support only `.`